charts/pmm-ha/templates/haproxy-configmap.yaml

```diff
@@ -15,6 +15,17 @@ data:
         log stdout    local2 info
         daemon
     
+    resolvers k8s
+        nameserver dns1 kube-dns.kube-system.svc.cluster.local:53
+        resolve_retries 3
```

Comment:
Seems like it could replace the whole haproxy-init-configmap if we increase the retries value